service/src/test/java/bio/terra/cbas/pact/VerifyPactsAllControllers.java

```diff
@@ -1,4 +1,4 @@
-package bio.terra.cbas.controllers;
+package bio.terra.cbas.pact;
```

Comment:
I'm creating a new pact package, rather than keeping this in the controllers package.

Why: I think we missed the mark by grouping this test with the controllers. I'm realizing that we should probably be involving more of the application code during this test.

If anyone disagrees, I can revert this - it's not important for the ECM pact test.